src/components/executive-education-2u/styles/_ExecutiveEducation2UErrorPage.scss

```diff
@@ -0,0 +1,33 @@
+.executive-education-2u-error-illustration {
+    align-items: center;
+    margin-bottom: 32px;
+}
+
+.executive-education-2u-error-heading {
+    align-items: center;
+    line-height: 60px;
+    margin-bottom: 32px;
+
+    .executive-education-2u-error-heading-red {
+        font-size: 60px;
+        font-weight: 700;
+        color: #D23228;
+        margin-right: 10px;
+    }
+
+    .executive-education-2u-error-heading-black {
+        font-size: 60px;
+        font-weight: 700;
+        color: #002121;
```

Comment:
Use variables (in this case, `$primary-700`) instead of hardcoding HEX values, which is an anti-pattern.